README.md

```diff
@@ -89,3 +89,25 @@ $ git tag -d v1.2.3                 # delete tag v1.2.3 locally
 $ git push origin v1.2.3            # push tag v1.2.3
 $ git push -d origin v1.2.3         # delete tag v1.2.3 remotely
 ```
+
+## Design patterns
+
+#### JS concepts
+
+- The repo is set up to import individual components, which then in turn import their own scss modules. 
+- Preference for hooks and functional components over classes.
+- Preference for proptypes validation over typescript and .d.ts files. 
```

Comment:
At this point I'd say not having .d.ts files is a bit unusual for a js library. I'm not trying to make a statement here, just saying that we prefer proptype validation.